### DISCUSSION TITLE
Outline v0.73.1 Installation from source - Empty page

### DISCUSSION THREAD

**Discussion by Termin0tor** [outline/outline#6191] [Self Hosting]
After the source installation of v.0.73.1 and setting up the nginx reverse proxy the website does not load anymore. It only shows an empty page.

**To Reproduce**

1. Install outline from source
2. Use nginx as a reverse proxy with these settings: https://docs.getoutline.com/s/hosting/doc/nginx-6htaRboR57

**Expected behavior**
Website loading properly

**Screenshots**

![grafik](https://github.com/outline/outline/assets/140089621/a4572980-ef4c-40c2-9ebb-a15cf8e7ac76)
**Outline:**
- Install: From source, selfhosted
- Version: v.0.73.1

**Desktop**
All devices are affected

**Comment by tommoor** [comment]
Not sure, could be many things given the information – however it looks like you're running in development mode which is not good. You should set `NODE_ENV=production`

**Comment by fuzyll** [comment]
I just had this problem while setting up a new development environment (#6193) and it turned out that some content was being blocked due to CORS, probably due to my not having an SSL certificate that my browser would accept.

Out of curiosity: Can you verify that the cert is trusted for wherever it's loading e.g. `index.tsx`? For me, this was being served up over port 3001 instead of 3000, which is why CORS got involved at all. Another thing to try might be setting this up via HTTP and seeing if that works, which might help narrow this down to the SSL certificate.

Of course, your problem might also be completely different. But, hopefully this helps someone.

**Comment by tongyin002** [comment]
Also having the same issue even starting up locally. I tried clearing browser HSTS but no luck. Did you happen to find a solution?

** somehow got it to load but stuck in auth error. I did configure AZURE so not sure why it does't go through
<img width="1460" alt="image" src="https://github.com/outline/outline/assets/34780527/99d296fd-0483-478b-89af-f64cdc2f7c2b">
